website/content/en/docs/building-operators/golang/installation.md

```diff
@@ -9,7 +9,7 @@ Follow the steps in the [installation guide][install-guide] to learn how to inst
 ## Additional Prerequisites
 
 - [git][git_tool]
-- [go][go_tool] version v1.13+.
```

Comment:
This version should stay on 1.13 until the scaffolds have been updated to use 1.15, right?